org.eclipse.scanning.test/src/org/eclipse/scanning/test/epics/EpicsV4ConnectorTest.java

```diff
@@ -237,19 +246,19 @@ public void connectToValidDeviceButOfflineWhenConfigure() throws Exception {
 					new SpiralModel("stage_x", "stage_y", 1, new BoundingBox(0, -5, 8, 3)), regions);
 			IPointGenerator<?> scan = pgService.createCompoundGenerator(temp);
 
-			EPICSv4ExampleModel pmac1 = new EPICSv4ExampleModel();
+			MalcolmModel pmac1 = new MalcolmModel();
 			pmac1.setExposureTime(23.1);
 			pmac1.setFileDir("/TestFile/Dir");
 
 			// Set the generator on the device
 			// Cannot set the generator from @PreConfigure in this unit test.
-			((AbstractMalcolmDevice<?>) modelledDevice).setPointGenerator(scan);
+			((AbstractMalcolmDevice<?>) malcolmDevice).setPointGenerator(scan);
 
 			epicsv4Device.stop();
 
 			try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)